rmw_connext_shared_cpp/src/create_topic.cpp

```diff
@@ -40,14 +40,11 @@ rmw_connext_shared_cpp::create_topic(
 
   std::lock_guard<std::mutex> guard(node_info->topic_creation_mutex);
 
-  DDS::TopicDescription * topic_description =
-    participant->lookup_topicdescription(dds_topic_name);
-
-  DDS::Topic * topic = nullptr;
-  DDS::ReturnCode_t status = DDS::RETCODE_ERROR;
-  if (!topic_description) {
+  DDS::Duration_t timeout = DDS::Duration_t::from_seconds(0);
```

Comment:
It isn't clear to me from the Connext API docs; does 0 mean "don't wait at all", or "wait forever"?  I'm presuming the former, but I just want to be sure.

Reply:
It means "don't wait at all".